webpack.config.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 module.exports = env=> {
-    return [
-        require('./config/webpack-dev.config'),
-        require('./config/webpack-prod.config')
-    ];
-};
\ No newline at end of file
+  return [
+    require(`./config/webpack-${env}.config`),
```

Comment:
I didn't understand why running webpack used 2 configurations. especially if I am just trying to do `webpack-dev-server --env dev` (which is what gets called with `npm start`) If you have other suggestions to get this working locally I am open to them. this is just the path of least change I found